### ISSUE TITLE
Feedback: Make link styling more prominent 

### ISSUE THREAD

**Issue by mariamarrero** [18F/govt-wide-patternlibrary#132]
I am afraid that he style of the "default link for body copy" may not be noticeable enough (dotted line). The standard style is the solid underline and people are used to this. When using style links, it is also important to remember that users don’t read; they scan. The links should be obvious. Perhaps the "hover" state can have a background color or become bold. 

We may also want to add a "Visited Links" section here.

**Comment by Scotchester** [comment]
Related: I came across this [page about links' contrast with surrounding text color and colorblindness](http://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/working-examples/G183/link-contrast.html). It recommends that the default link color have at least a 3:1 contrast ratio with the surrounding text color, and that something in the blue family be used to avoid creating problems for the most common type of colorblindness. Food for thought as decisions about color palette are made.

**Comment by Scotchester** [comment]
One word of caution about the idea of making text bold on hover: This will shift text position as the text expands on mouseover and then contracts back to what it was on mouseout.

**Comment by colinpmacarthur** [comment]
As Maria points out, what's most important is that links look "clickable." However, there's some debate over whether underlining them does that. Some relatively recent research on this topic (http://www.nngroup.com/articles/clickable-elements/) doesn't come down heavily on one side or the other. Some older academic research found underlined links actually reduce readability (https://vsis-www.informatik.uni-hamburg.de/getDoc.php/publications/119/www2003.pdf). 

I think this is an ideal candidate for usability testing.

**Comment by colinpmacarthur** [comment]
Thanks for your thoughts, all! I've added this to the project workflow.
